packages/media/src/types/resource.ts

```diff
@@ -77,20 +77,13 @@ export interface Resource {
   /** Whether the resource externally hosted. */
   isExternal: boolean;
   /** Whether the resource is a placeholder. */
-  isPlaceholder?: boolean;
+  isPlaceholder: boolean;
   /** Whether the resource needs a CORS proxy. */
-  needsProxy?: boolean;
```

Comment:
IIRC these were made optional because especially in templates not all resources have this property. I think we only added in later and didn't add a data migration because it was kinda unnecessary.

So to reflect that, we should keep them optional, or add a migration.